src/components/organisms/leporello-graphic-details-item/leporello-graphic-details-item.jsx

```diff
@@ -103,89 +102,90 @@ export default ({
       closerType="with-cover"
     >
       <div className={`leporello-graphic-details-item ${additionalClassNames.join(' ')}`}>
-        <div className={'leporello-graphic-details-item__image'}>
-          <Image
-            src={images.sizes.l.src}
-            alt={title}
-            modifierWithBox={false}
-          />
-        </div>
+        <header className="leporello-graphic-details-item__header">
 
-        <div className="leporello-graphic-details-item__info">
-          <h1 className="title">{ title }</h1>
-          <h2 className="subtitle inventor">
-            { subtitle }
+          <h1 className="title">{title}</h1>
+          <h2 className="subtitle">
+            {classification.classification}, {objectName}
           </h2>
-
-          <div className="further-infos">
-
-            <DefinitionList>
-
-
-              <DefinitionList.Entry
-                term={ t('Attribution') }
-                definition={ <InvolvedPersonsList data={ involvedPersons } /> }
-              />
-
-              <DefinitionList.Entry
-                term={ t('Classification') }
-                definition={ `${classification.classification}, ${objectName}` }
-              />
-              <DefinitionList.Entry
-                term={ t('Dating') }
-                definition={ `${dating.dated} ${dating.remarks}` }
-              />
-              <DefinitionList.Entry
-                term={ t('Measurments') }
-                definition={dimensions}
-              />
-              { signature && <DefinitionList.Entry
-                term={ t('Signature') }
-                definition={signature}
-              />
-              }
-
-              { inscription
-                && <DefinitionList.Entry
-                term={ t('Inscription') }
-                definition={inscription}
-              />
-              }
-
-              <DefinitionList.Entry
-                term="CDA ID"
-                definition={inventoryNumber}
-              />
-              <DefinitionList.Entry
-                term={ t('Persistent Link') }
-                definition={location}
-              />
-              {
-                catalogWorkReferenceItems.map(
-                  item => (
+        </header>
+
+        <div className="leporello-graphic-details-item__content">
+          <div className={'leporello-graphic-details-item__content-image'}>
+            <Image
+              src={images.sizes.l.src}
+              alt={title}
+              modifierWithBox={false}
```

Comment:
Wäre `false` ein möglicher Default-Value für `modifierWithBox`?

Wenn ja, könnte es Sinn machen die Zeile [src/components/atoms/image/image.jsx:10](https://github.com/lucascranach/cranach-grafiken-gatsby/blob/3e81503291d4c7f174cee23b7ebbd6e991d1ad73/src/components/atoms/image/image.jsx#L10) von `modifierWithBox,` in `modifierWithBox = false,` zu ändern. Dann müsste man für die Verwendung Image-Komponente nicht immer jedes Mal ein Wert für `modifierWithBox` angeben.

Ein Default-Wert könnte aber problematisch werden, wenn sich dieser über die Zeit ändert.